app/serializers/answer_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class AnswerSerializer < ActiveModel::Serializer
+  attributes :id, :response, :user_id, :accepted, :cached_votes_up, :cached_votes_down
```

Comment:
@jerflan - rather than user_id in comments, answers and questions, can you pass the first and last name? You can do this similar the way you added comments and answers to the questions serializers.